docs/installation/frameworks/overview.md

```diff
@@ -6,11 +6,17 @@ menu-title: Overview
 
 # Integrating CKEditor 5 with JavaScript frameworks
 
+<info-box hint>
+**Quick recap**
+
+In the previous guides you have learned several different methods of getting you CKEditor 5 installation up and running. Now you will learn how to utilize the power of Java Script framework to include your editor as a component.
```

Comment:
`Java Script` -> `JavaScript`

Also, this sentence sounds a bit weird to me:

> Now you will learn how to utilize the power of Java Script framework to include your editor as a component

I think I understand what you mean here, but maybe we can rephrase it and emphasise that we have ready-to-use CKEditor 5 components for Angular, React and Vue?

Reply:
Fixed in commit.